src/service/action-impl.js

```diff
@@ -733,22 +735,39 @@ export class ActionService {
   }
 }
 
-
 /**
  * Returns `true` if the given action invocation is whitelisted in the given
- * whitelist.
+ * whitelist. Default actions' alias, 'activate', are automatically
+ * whitelisted if their corresponding registered alias is whitelisted.
  * @param {!ActionInvocation} invocation
  * @param {!Array<{tagOrTarget: string, method: string}>} whitelist
  * @return {boolean}
  * @private
  */
 function isActionWhitelisted_(invocation, whitelist) {
+  const {method, node, tagOrTarget} = invocation;
+  const tagOrTargetRegex = caseInsensitiveMatch(tagOrTarget);
+  let invocationMethodRegex = caseInsensitiveMatch(method);
+  const defaultActionAlias = node.getDefaultActionAlias();
+  if (method == DEFAULT_METHOD && (node
+      && defaultActionAlias)) {
+    invocationMethodRegex = caseInsensitiveMatch(defaultActionAlias);
+  }
+
   return whitelist.some(({tagOrTarget, method}) => {
-    return (tagOrTarget === '*' || tagOrTarget === invocation.tagOrTarget) &&
-        method === invocation.method;
+    return (tagOrTarget === '*' || tagOrTargetRegex.test(tagOrTarget)) &&
+        (invocationMethodRegex.test(method));
   });
 }
 
+/**
+ * Generate a case insensitive regex with the provided string.
+ * @param {string} toMatch
+ * @return {RegExp}
+ */
+function caseInsensitiveMatch(toMatch) {
+  return new RegExp('^' + toMatch + '$', 'i');
+}
```

Comment:
Minor: Not a fan of using regex for case-insensitive compare. Would prefer `toLowerCase()` and equality check.